app/upgrades/v5/upgrades.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/tharsis/evmos/v5/types"
 	claimskeeper "github.com/tharsis/evmos/v5/x/claims/keeper"
 	claimstypes "github.com/tharsis/evmos/v5/x/claims/types"
+
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
```

Comment:
nit, group with sdk pkgs

Reply:
done in [b5bfcce](https://github.com/tharsis/evmos/pull/699/commits/b5bfcce1069325b3d44cb512d3b8dff0bf192a1a)